R/geom-path.r

```diff
@@ -302,7 +302,7 @@ GeomStep <- ggproto("GeomStep", GeomPath,
 #
 # @keyword internal
 stairstep <- function(data, direction="hv") {
-  direction <- match.arg(direction, c("hv", "vh"))
+  direction <- match.arg(direction, c("hv", "vh", "hvh"))
```

Comment:
Maybe "mid" ? I think "hvh" implies that we might also add "vhv"

Reply:
"mid" makes sense, "vhv" wouldn't. I have made the change.